src/Core/Core.csproj

```diff
@@ -11,12 +11,9 @@
 
     <RequiresNewtonsoft>true</RequiresNewtonsoft>
     <RequiresSystemIOAbstractions>true</RequiresSystemIOAbstractions>
+    <RequiresChannels>true</RequiresChannels>
```

Comment:
Our convention for some packages that are references in more than one place, to make sure we're referencing the same version.

Reply:
Does `Core` still need channels?

Reply:
Ah - yes, because the events are still in `Core`. Could they be in `ConnectedMode`?

Reply:
I will revisit once I implement the Taint consumer